### PR TITLE
Feat/todo-app

### PR DESCRIPTION
**Pull request by tobidechamp15**
I have created the todo list 
PERMISSION TO MERGE? 
@peterdgreat

### CODE REVIEW

**Comment by peterdgreat** [comment]
> I have created the todo list PERMISSION TO MERGE? @peterdgreat

Will check now

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (peterdgreat) [inline README.MD:54]: Hmm, seems you used my template here. Nice one✌️

File: style.css [hunk 2]
```diff
@@ -7,12 +102,14 @@ li {
...
   color: #ffffff;
   font-size: 20px;
   font-weight: 500;
   font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
     "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+  text-align: center;
+  overflow: scroll;
 }
 body {
   /* height: 100vh; */
```
Comment (peterdgreat) [inline style.css:115]: avoid leaving unused codes

File: style.css [hunk 3]
```diff
@@ -54,49 +151,21 @@ h1 {
   height: 35px;
   border: none;
   border-radius: 20px;
-  padding: 10px;
+  /* padding: 10px; */
```
Comment (peterdgreat) [inline style.css:154]: same here

File: style.css [hunk 4]
```diff
@@ -54,49 +151,21 @@ h1 {
...
   background-color: rgb(246, 245, 244);
   outline: none;
   color: #283ccf;
-  font-size: 24px;
+  /* font-size: 24px; */
```
Comment (peterdgreat) [inline style.css:158]: here too

**Review by peterdgreat — APPROVED** [review]
Everything here is good to merge

File: style.css (hunk 4 above)
Comment (tobidechamp15) [reply style.css:158]: Yes sir

**Pull request merged**